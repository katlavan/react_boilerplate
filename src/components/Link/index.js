import React from 'react';

const Link = ({url, title, text}) => <a href={url} title={title}>{text}</a>;

export default Link;