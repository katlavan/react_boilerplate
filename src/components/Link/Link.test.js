import React from 'react';
import Link from './';
import renderer from 'react-test-renderer';

it('Link renders correctly', () => {
  const tree = renderer
    .create(<Link
      url='https://instagram.com'
      title='Go to instagram.com'
      text='Instagram'
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});