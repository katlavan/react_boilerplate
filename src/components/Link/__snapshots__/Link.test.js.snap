// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<a
  href="https://instagram.com"
  title="Go to instagram.com"
>
  Instagram
</a>
`;
